scripting/include/srccoop/playerpatch.inc

```diff
@@ -6,6 +6,8 @@
 #pragma newdecls required
 #pragma semicolon 1
 
+static float g_pvec3PlayerShootPosition[MAXPLAYERS][3];
```

Comment:
[MAXPLAYERS + 1] and probably unintentional p in variable name

Reply:
Yeah that was unintentional. Fixed the array size and removed `p` from the variable name.